### ISSUE TITLE
Some doc examples seem to behave with new infrastructure 

### ISSUE THREAD

**Issue by sofianehaddad** [openturns/openturns#1794]
Hi all,

In the new documentation, it seems that we can not control the example output. We only assess that example runs (no fail)

For example, let us have a look at the `estimate of a multivariate distribution` example. The focus is to highlight the handling of (potential) `composed copula`. For that purpose we define an example with some `marginal distributions` and a `composed copula`. The objective is to illustrate how we can split data onto (independent) blocks.

The example illustrated in `openturns` `1.15` with the oldest technology (`jupyter-notebook`) is available [here](https://openturns.github.io/openturns/1.15/examples/data_analysis/estimate_multivariate_distribution.html)

Within the new techno (python script), example is available [here](https://openturns.github.io/openturns/latest/auto_data_analysis/distribution_fitting/plot_estimate_multivariate_distribution.html)

If we focus on bloc estimate (for copula), output was 
![image](https://user-images.githubusercontent.com/1433220/110846637-4f382f00-82ac-11eb-86a6-94e6287031e0.png)

in accordance with fixed model. Now we got 
![image](https://user-images.githubusercontent.com/1433220/110846703-60813b80-82ac-11eb-95a0-1ffa79c4558a.png)

In other words, we lost all the interest of the example. 

As I can not reproduce the result of `1.16` I am wondering how/why do we get such result?

**Comment by josephmure** [comment]
I guess that the sample produced when sphinx-gallery ran the script must have had a different Spearman correlation matrix than the one produced when the notebook was originally run. In order to get reproducible results, we need to set the random seed.

**Comment by jschueller** [comment]
exactly, examples seem to run in the same context, so we might need to set the seed
this is the same behavior as the docstrings
